src/Symfony/Component/Routing/Tests/Fixtures/AnnotationFixtures/InvokableMethodController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AnnotationFixtures;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+class InvokableMethodController
+{
+    /**
+     * @Route("/here", name="lol", methods={"GET", "POST"}, schemes={"https"})
```

Comment:
Suggestion: remove the annotation and provide only an attribute. See:

- #49369

Reply:
I would keep this test case, no need to degrade the DX, even if we plan to deprecate annotations.

Reply:
anyway, we cannot remove this annotation just for this new case, as this is the _existing_ annotation.

Reply:
Yes I misread this as a new annotation, sorry.